tensorbay/client/job.py

```diff
@@ -117,9 +124,11 @@ def update(self, until_complete: bool = False) -> None:
 
     def abort(self) -> None:
         """Abort a :class:`Job`."""
+        self._client.open_api_do("POST", f"jobs/{self.job_id}/abort", self._dataset_id)
```

Comment:
Do we need to call the `update()` here?

Reply:
In my opinion, the default actions should be reduced.

Reply:
abort and retry method MUST change the status of a Job, which means the Job status will be outdated right after user calling `abort` and `retry`.
I do not know should us update the status for users here. I prefer to update for them since it is not the same as the `checkout` method. 
But of course we can reduce the default action here, and wait user tell us which way is better.